src/generic/generic.service.ts

```diff
@@ -1,19 +0,0 @@
-import { Injectable, signal } from '@angular/core';
-import { FormBuilder, FormGroup } from '@angular/forms';
-
-@Injectable({
-  providedIn: 'root',
-})
-export class GenericService {
-  // form = new Subject<FormGroup>(); // alternate solution
-  emptyForm: FormGroup = this.fb.group({});
-  // which form is active (ex: patient form or prescriber form)
-  form = signal<FormGroup>(this.emptyForm);
```

Comment:
Was storing signals in the service the way that the course showed this?

Reply:
In the course they did use computed signals that were stored in the service. Their use case was a little different because they were storing product info from an http request instead of a form but I think it's kind of comparable, I included screenshots of the course code below:
<img width="709" alt="Screenshot 2024-03-29 at 12 38 45" src="https://github.com/mielah/stackblitz-starters-dajx6h/assets/17678552/3aa8b3fe-bbfe-46b2-91bd-2cd696da51a7">
<img width="680" alt="Screenshot 2024-03-29 at 12 39 05" src="https://github.com/mielah/stackblitz-starters-dajx6h/assets/17678552/1b8274aa-ceb7-467b-bf9e-e23690a87cb7">